apps/ledger-live-mobile/src/screens/PostOnboarding/PostOnboardingHub.tsx

```diff
@@ -0,0 +1,194 @@
+import React, { useCallback, useEffect, useState } from "react";
+import { Box, Flex, Icons, Log, Notification, Text } from "@ledgerhq/native-ui";
+import { useTranslation } from "react-i18next";
+import { ScrollView, StyleSheet } from "react-native";
+import { SafeAreaView } from "react-native-safe-area-context";
+import styled from "styled-components/native";
+import { useFocusEffect, useNavigation } from "@react-navigation/native";
+import Animated, {
+  interpolate,
+  runOnJS,
+  useAnimatedStyle,
+  useSharedValue,
+  withDelay,
+  withTiming,
+} from "react-native-reanimated";
+import { StackScreenProps } from "@react-navigation/stack";
+import {
+  useAllPostOnboardingActionsCompleted,
+  useClearLastActionCompletedCallback,
+  usePostOnboardingHubState,
+} from "../../logic/postOnboarding/hooks";
+import PostOnboardingActionRow from "../../components/PostOnboarding/PostOnboardingActionRow";
+import { NavigatorName } from "../../const";
+
+const SafeContainer = styled(SafeAreaView).attrs({
+  edges: ["left", "bottom", "right"],
+})`
+  flex: 1;
+`;
+
+const Divider = styled(Box).attrs({
+  height: 0.5,
+  backgroundColor: "neutral.c30",
+  width: "100%",
+})``;
+
+const AnimatedFlex = Animated.createAnimatedComponent(Flex);
+
+const PostOnboardingHub: React.FC<StackScreenProps<{}>> = ({ navigation }) => {
+  const { t } = useTranslation();
+  const { lastActionCompleted, actionsState } = usePostOnboardingHubState();
+  const [popupOpened, setPopupOpened] = useState(true);
+  const { actionCompletedHubTitle, actionCompletedPopupLabel } =
+    lastActionCompleted ?? {};
+
+  const clearLastActionCompleted = useClearLastActionCompletedCallback();
+
+  useEffect(
+    () => () => {
+      /**
+       * The last action context (specific title & popup) should only be visible
+       * the 1st time the hub is navigated to after that action was completed.
+       * So here we clear the last action completed.
+       * */
+      clearLastActionCompleted();
+    },
```

Comment:
I think technically you could just do
```
useEffect(()=>clearLastActionCompleted, [clearLastActionCompleted]);
```
since the returned value of the `useEffect` is a function and it will be called on onmount 🤔

Reply:
since we don't pass parameters, if we did it wouldn't work

Reply:
Makes sense, thanks!